### PR TITLE
Restore default placeholder for integer input with base 0

### LINKED ISSUE #10725: Integer input: placeholder for base 0
In #6980, a custom placeholder was added. Previously, the placeholder would be "integer" if base was 10, or "integer in base XX" for other bases. This was kept as default, but previously, for base=0 (i.e., accept 0x/0b/0o prefixes), the default placeholder would be "integer", but it now shows as "integer in base 0", which can be confusing for students. I think "integer" should be the default placeholder in these cases.

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Fixes #10725. Restores the default placeholder from before #6980 for cases with base zero (i.e., base 10 with option of using a different base with a corresponding prefix).

### CODE REVIEW

File: apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache [hunk 1]
```diff
@@ -107,7 +107,7 @@
 {{#format}}
 <p>
 Your answer must be an integer{{^default_base}} in base {{base}}{{/default_base}}{{#zero_base}} in base 10. You may also use an integer in base 16 with the prefix <code>0x</code> (e.g., 0x1a), in base 2 with the prefix <code>0b</code> (e.g. 0b1101), or in base 8 with the prefix <code>0o</code> (e.g., 0o777){{/zero_base}}.
-It must not contain a decimal, though it may contain underscores (which are ignored).
+It must not contain a decimal point, though it may contain underscores (which are ignored).
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache:110]: This is not related, it's just changed for clarity.

File: apps/prairielearn/elements/pl-integer-input/pl-integer-input.py [hunk 2]
```diff
@@ -111,7 +111,9 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             placeholder = pl.get_string_attrib(element, "placeholder")
         else:
             placeholder = (
-                "integer" if base == BASE_DEFAULT else f"integer in base {base}"
+                "integer"
+                if base == BASE_DEFAULT or base == 0
+                else f"integer in base {base}"
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-integer-input/pl-integer-input.py:116]: This was the logic before #6980:
https://github.com/PrairieLearn/PrairieLearn/blob/ce7799a7aa6ed401ace122132fb040789d439817/apps/prairielearn/elements/pl-integer-input/pl-integer-input.py#L88
https://github.com/PrairieLearn/PrairieLearn/blob/ce7799a7aa6ed401ace122132fb040789d439817/apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache#L111-L113

**Review by eliotwrobson — APPROVED** [review]
Seems reasonable 👍🏽
